app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -62,9 +62,8 @@
 <% end %>
 
 <%= render 'two_factor_authentication/troubleshooting_options', presenter: @presenter %>
-
-<% if MfaPolicy.new(current_user).two_factor_enabled? %>
+<% if MfaPolicy.new(current_user).two_factor_enabled? && !in_multi_mfa_selection_flow? %>
```

Comment:
I'm hadn't necessarily expected that the "Cancel" text needed to change, more the link destination. In that regard, I don't know that we need to actually update the view here, and instead look at fixing the logic of `PhoneDeliveryPresenter#cancel_link` to handle the scenario where someone in the multi-MFA selection flow should be returned to the MFA selection screen.

https://github.com/18F/identity-idp/blob/a7ed678f01dcd5c3d38bf64fc8ccd1cff9769a95/app/presenters/two_factor_auth_code/phone_delivery_presenter.rb#L59-L66

Reply:
I see. I took the title too literally. I'll work it back from the presenter instead.